include/psabpf.h

```diff
@@ -243,14 +243,18 @@ typedef struct psabpf_match_key {
         } range;
     } u;
 
-    bool is_data_owner;
+    /* Used to tell whether instance is weak copy of another instance. In such case
```

Comment:
My proposition is: 
`/* Used to tell whether this psabpf_match_key_t instance can be freed or not. If True then this instance memory can be freed. Otherwise not. .... explanation why...
 */`